ktor-client/ktor-client-apache/jvm/src/io/ktor/client/engine/apache/ApacheEngine.kt

```diff
@@ -58,11 +58,11 @@ internal class ApacheEngine(override val config: ApacheEngineConfig) : HttpClien
             disableAuthCaching()
             disableConnectionState()
             disableCookieManagement()
-            setDefaultIOReactorConfig(IOReactorConfig.custom().apply {
-                setMaxConnPerRoute(MAX_CONNECTIONS_COUNT)
-                setMaxConnTotal(MAX_CONNECTIONS_COUNT)
-                setIoThreadCount(IO_THREAD_COUNT_DEFAULT)
-            }.build())
+            setMaxConnPerRoute(MAX_CONNECTIONS_COUNT)
+            setMaxConnPerRoute(MAX_CONNECTIONS_COUNT)
```

Comment:
You have it twice, should be `setMaxConnTotal`

Reply:
No idea how I've messed that one up. Thanks for the catch.